Tests/Command/DropDatabaseDoctrineTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+
+namespace Doctrine\Bundle\DoctrineBundle\Tests\Command;
+
+use Doctrine\Bundle\DoctrineBundle\Command\DropDatabaseDoctrineCommand;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class DropDatabaseDoctrineTest extends TestCase
+{
+
+    public function testExecute()
+    {
+        $connectionName = 'default';
+        $dbName = 'test';
+        $params = array(
+            'url' => "sqlite:///". sys_get_temp_dir() ."/test.db",
+            'path' => sys_get_temp_dir() . "/" . $dbName,
+            'driver' => 'pdo_sqlite',
+        );
+
+        $application = new Application();
+        $application->add(new DropDatabaseDoctrineCommand());
+
+        $command = $application->find('doctrine:database:drop');
+        $command->setContainer($this->getMockContainer($connectionName, $params));
+
+        $commandTester = new CommandTester($command);
+        $commandTester->execute(
+            array_merge(array('command' => $command->getName(), '--force' => true))
+        );
+
+        $this->assertContains("Dropped database for connection named " . sys_get_temp_dir() . "/" . $dbName . "" , $commandTester->getDisplay());
+    }
+
+    /*
```

Comment:
If this test is not needed, can you remove it instead of leaving it commented out?